src/main/java/jeresources/util/MobTableBuilder.java

```diff
@@ -12,7 +12,7 @@
 import java.util.Map;
 
 public class MobTableBuilder {
-    private final Map<ResourceLocation, LivingEntity> mobTables = new HashMap<>();
+    private final Map<LivingEntity, ResourceLocation> mobTables = new HashMap<>();
```

Comment:
Having an instance of LivingEntity is a good map key in my opinion.
Why did you swap them?

Reply:
I originally tried to sort the mobs in the mob view by swapping key and value in the map, hoping that Java would automatically sort the LivingEntities by their names. That didn't work, and I got it to work with the `.sorted(...)` line in `registerVanillaMobs`. I can of course revert the swapping if you want it, as it is now redundant

Reply:
Please revert to having the ResourceLocation as key here